eosdis-nasa/earthdata-pub-overview
# Fixes daac dropdown

# Description

Fixes the issue where the DAAC dropdown wasn't visible

## Spec

See Ticket: https://bugs.earthdata.nasa.gov/browse/EDPUB-1409

---

## Validation

1. Make sure all merge request checks have passed (CI/CD).
2. Pull related branches locally.
3. Navigate to http://localhost:8082/
4. Click the Find A DAAC dropdown
5. Verify that the dropdown selections are visible.
